### PR TITLE
fix codespell typos

### PR DESCRIPTION
**Pull request by gemesa**
Fixed all issues reported by [codespell](https://github.com/codespell-project/codespell). #2500 is a follow-up of this PR.

Most issues were comment, readme, manpage, print message etc. typos but there were a couple in the code:

`lib/csharp/WirelessPanda/*.cs`:

I verified on my windows machine the code did not break:

```
C:\Users\andras\Downloads\aircrack-ng-codespell\lib\csharp\WirelessPanda>C:\Windows\Microsoft.NET\Framework64\v4.0.30319\csc.exe /target:library Readers\csvReader.cs Readers\kismetCsvReader.cs Readers\NetXMLReader.cs Readers\Reader.cs Readers\UniversalReader.cs AccessPoint.cs Coordinates.cs Station.cs WirelessDevice.cs Properties\AssemblyInfo.cs
Microsoft (R) Visual C# Compiler version 4.8.4084.0
for C# 5
Copyright (C) Microsoft Corporation. All rights reserved.

This compiler is provided as part of the Microsoft (R) .NET Framework, but only supports language versions up to C# 5, which is no longer the latest version. For compilers that support newer versions of the C# programming language, see http://go.microsoft.com/fwlink/?LinkID=533240

Readers\UniversalReader.cs(99,21): warning CS0162: Unreachable code detected

C:\Users\andras\Downloads\aircrack-ng-codespell\lib\csharp\WirelessPanda>

```

Codespell config is stored in `.codespellrc`.
The following files are excluded:
- `*.patch`
- `./LICENSE.OpenSSL`
- `./test/password.lst`
- `./test/password-2.lst`
- `./test/password-3.lst`

The following words are excluded:
- `aci` (`WME_PARAM_ACI`)
- `ba` (`sed ':a;N;$!ba;s/\n/ /g'`)
- `curent` (`struct dirent * curent;`)
- `datas` (`unsigned char * datas;` here `data` is already taken)
- `inout` (`InOut`)
- `keep-alives` (`seconds between keep-alives`)
- `nnumber` (`nNumber`)
- `overthere` (`OverThere` as a username)
- `parm` (used a lot in osdep so must be on purpose)
- `parms` (used a lot in osdep so must be on purpose)
- `ue` (`except URLError as ue:`)
- `varian` (`#define	ETHERTYPE_VARIAN	0x80DD	/* Varian Associates */`)
- `wan` (`WAN`)

### CODE REVIEW

**Comment by Mister-X-** [comment]
I know it may be a lot of work, but would it be possible to split in 2 PR:
1. Add CodeSpell CI
2. Fix spelling

**Comment by gemesa** [comment]
> I know it may be a lot of work, but would it be possible to split in 2 PR:
> 
>     1. Add CodeSpell CI
> 
>     2. Fix spelling

sure no problem

**Comment by gemesa** [comment]
I split it into 2 PRs, please review them.

File: test/int-test-common.sh [hunk 1]
```diff
@@ -155,7 +155,7 @@ backup_regdomain() {
 set_regdomain() {
 	if [ -z "$1" ]; then
 		echo "set_regdomain(): No regdomain given"
-		retun 1
+		return 1
```
Comment (Mister-X-) [inline test/int-test-common.sh:158]: I'm quite puzzled to see that it has been there for quite a while before being noticed. Is this something never included anywhere?

**Pull request merged**